packages/runtime/src/types.ts

```diff
@@ -407,3 +407,33 @@ export enum Base64Encoding {
 	URL = 0,
 	STD = 1,
 }
+
+export interface BaseTxnReceipt {
```

Comment:
You dont need copy it, just import it from @algo-builder/algob package I think

Reply:
Actually the type in both are little different, algob TxnReceipt is a subset of runtime type i.e runtime has much broader keys available, that's why added it